src/main/java/ch/njol/skript/expressions/ExprBlockData.java

```diff
@@ -29,7 +30,7 @@
 public class ExprBlockData extends SimplePropertyExpression<Object, BlockData> {
 
 	static {
-		String types = Skript.isRunningMinecraft(1, 19, 4) ? "blocks/displays" : "blocks";
+		String types = Skript.isRunningMinecraft(1, 19, 4) ? "blocks/displays/entities" : "blocks/entities";
 		register(ExprBlockData.class, BlockData.class, "block[ ]data", types);
```

Comment:
```suggestion
		register(ExprBlockData.class, BlockData.class, "block[ ]data", "blocks/displays/entities");
```
This check can be removed since Skript now only supports 1.20.4+